kumascript/macros/EmbedLiveSample.ejs

```diff
@@ -10,16 +10,16 @@
 //  $5 - (Deprecated) The class name of the frame; defaults to "sample-code-frame". If you
 //       pass this parameter and give it a value other than "sample-code-frame",
 //       then the "Open in CodePen"/"Open in JSFiddle" buttons will not be displayed.
-//  $6 - (Deprecated) Allowed features, separated by semicolons (optional)
+//  $6 - Allowed features, separated by semicolons (optional)
```

Comment:
I had to look up what "allowed features" means, could we give one example (`geolocation` maybe?) and what it defaults to?

Reply:
```suggestion
//  $6 - Allowed features (like geolocation, microphone), separated by semicolons (optional).
//      Defaults to ''.
```